src/v/coproc/ntp_context.h

```diff
@@ -30,8 +30,8 @@ namespace coproc {
 /// subscribers to an input ntp
 struct ntp_context {
     struct offset_pair {
-        model::offset last_read{model::model_limits<model::offset>::min()};
-        model::offset last_acked{model::model_limits<model::offset>::min()};
+        model::offset last_read{0};
```

Comment:
Why do we initialize with `0` ? Whole semantics of offsets in Redpanda is inclusive i.e. 0 means that record with offset `0` was already delivered to cooprocessor

Reply:
This is why I initially had the offset initialized to min. However when performing some tests, I observed that a newly created topic with no data has offset 0 for committed when querying the log stats.

Reply:
Maybe offset accounting is incorrect in other place ?

Reply:
ah right! You just reminded me about that first batch thats in every log. Ok , going to modify this